frontend/package.json

```diff
@@ -30,5 +30,6 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "packageManager": "yarn@1.22.19+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447"
```

Comment:
nit: I believe this might have been an automatic change an needs to be removed.

Reply:
Thanks for catching! I've been removing this over and over, but it keeps popping up when I switch branches. 😅